src/libraries/System.Utf8String.Experimental/src/System/Runtime/Intrinsics/Intrinsics.Shims.cs

```diff
@@ -8,6 +8,8 @@ internal static class Vector64
         public static Vector64<ulong> Create(ulong value) => throw new PlatformNotSupportedException();
         public static Vector64<uint> CreateScalar(uint value) => throw new PlatformNotSupportedException();
         public static Vector64<byte> AsByte<T>(this Vector64<T> vector) where T : struct => throw new PlatformNotSupportedException();
+        public static Vector64<T> GetLower<T>(this Vector128<T> vector) where T : struct => throw new PlatformNotSupportedException();
```

Comment:
These are not strictly needed for this PR, but all of these APIs will be needed in subsequent PRs. Let me know if you'd rather that I keep this clean.